experimental/segmenter/src/line.rs

```diff
@@ -99,7 +99,10 @@ impl Default for LineBreakOptions {
 pub struct LineBreakSegmenter {
     options: LineBreakOptions,
     payload: DataPayload<LineBreakDataV1Marker>,
-    dictionary_payload: DataPayload<UCharDictionaryBreakDataV1Marker>,
+    dictionary_km_payload: Option<DataPayload<UCharDictionaryBreakDataV1Marker>>,
```

Comment:
Request: Please make LineBreakSegmenter work with either dictionary or LSTM

Reply:
I will add `[cfg(feature = "lstm")]` or not.